Cargo.toml

```diff
@@ -38,3 +38,7 @@ benchmarks = []
 std = []
 default = ["serde", "std"]
 use-pkg-config = ["libsodium-sys/use-pkg-config"]
+
+[package.metadata.docs.rs]
+default-target = "x86_64-unknown-linux-gnu"
+rustc-args = ["-C target-feature=+aes", "-C target-feature=+pclmulqdq"]
```

Comment:
Wouldn't -C target-cpu=native be enough?

Reply:
To your earlier point doing it this way ensures that the docs will always be “expanded” regardless of what platform they’re built on. I could arbitrarily pick a target with AES/CMUL support but doing it this way documents intent. There’s an [open discussion](https://github.com/rust-lang/rust/issues/1998) about how rude docs should handle conditional compilation (feature detection or otherwise) but this seems like the lesser of evils for now.